client/java/src/main/java/io/openlineage/client/transports/KafkaConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.openlineage.client.transports;
+
+import java.util.Properties;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor
+@ToString
+public final class KafkaConfig implements TransportConfig {
+  @Getter @Setter private String topicName;
+  @Getter @Setter private String localServerId;
+  @Getter @Setter private String bootstrapServerUrl;
```

Comment:
Is there an option to provide multiple bootstrap servers? 
It's really useful to have such. 
It can be comma separated list of bootstrap servers.